rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -85,6 +85,7 @@
   private static final String KEY_ENABLE_FILE_STALESSNESS_CHECK = "rubix.enable.file.staleness-check";
   private static final String KEY_STALE_FILEINFO_EXPIRY_PERIOD = "rubix.stale.fileinfo.expiry.period";
   private static final String KEY_CLEANUP_FILES_DURING_START = "rubix.cleanup.files.during.start";
+  private static final String KEY_CACHE_DIRECTORY_REFRESH_TTL = "rubix.cache.dircetory.refresh.ttl";
```

Comment:
Nit: should be `..cache.directory.refresh..`

Reply:
Reminder to add documentation for new configuration option (if we are exposing this)